src/Illuminate/Queue/Console/WorkCommand.php

```diff
@@ -91,6 +93,21 @@ public function fire()
      */
     protected function runWorker($connection, $queue, $delay, $memory, $daemon = false)
     {
+        $timeout = $this->option('timeout');
+
+        if ($timeout > 0) {
+
+            if (! extension_loaded('pcntl')) {
```

Comment:
for example, on php-fpm, the extension gets loaded, but the methods are disabled.